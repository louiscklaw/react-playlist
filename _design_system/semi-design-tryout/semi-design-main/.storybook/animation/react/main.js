const config = require('../../base/base');

module.exports = {
  ...config,
  "stories": [
    '../../../packages/(semi-animation-react|semi-animation-styled)/_story/*.react.stories.(js|jsx)',
  ],
  typescript: {
    check: false,
    checkOptions: {}
  },
};
