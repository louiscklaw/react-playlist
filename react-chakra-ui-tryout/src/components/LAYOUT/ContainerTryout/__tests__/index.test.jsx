import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { ContainerTryout } from '../';

describe('test ContainerTryout tryout', () => {
  test('helloworld', () => {
    const tree = renderer.create(<ContainerTryout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
