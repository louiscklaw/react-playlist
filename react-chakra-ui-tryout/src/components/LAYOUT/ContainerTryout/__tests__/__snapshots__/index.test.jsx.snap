// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test ContainerTryout tryout helloworld 1`] = `
<div>
  ContainerTryout
  <div>
    <div
      className="chakra-container css-0"
    >
      There are many benefits to a joint design and development system. Not only does it bring benefits to the design team, but it also brings benefits to engineering teams. It makes sure that our experiences have a consistent look and feel, not just in our design specs, but in production
    </div>
  </div>
  <div>
    <div
      className="chakra-stack css-owjkmg"
    >
      <div
        className="chakra-container css-tflc3m"
      >
        Extra-Large Container
      </div>
      <div
        className="chakra-container css-f91va8"
      >
        Large Container
      </div>
      <div
        className="chakra-container css-gygvqc"
      >
        Medium Container
      </div>
      <div
        className="chakra-container css-15pnpc5"
      >
        Small Container
      </div>
    </div>
  </div>
  <div>
    <div
      className="chakra-container css-wa8fat"
    >
      <div
        className="css-rsv04h"
      >
        There are many benefits to a joint design and development system. Not only does it bring benefits to the design team.
      </div>
    </div>
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
</div>
`;
